src/main/java/com/robothand/highqualitybot/Utils.java

```diff
@@ -0,0 +1,38 @@
+package com.robothand.highqualitybot;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Hashtable;
+import java.util.Scanner;
+
+/**
+ * Static class that stores various utility methods and sundries.
+ */
+public class Utils {
+
+    // Loads the properties from a given .cfg file into a Hashtable
+    public static Hashtable<String,String> readCFG(String filename) throws FileNotFoundException {
+        Hashtable<String,String> properties = new Hashtable<>();
+        File file = new File(filename);
+
+        Scanner s = new Scanner(file);
+
+        while (s.hasNextLine()) {
+            String[] split = s.nextLine().split("=");
+            String key, value;
+
+            if (split.length < 2 || split[0].trim().startsWith("#")) {
+                continue;
+            }
+
+            key = split[0].trim();
+            value = split[1].trim();
+
+            if (properties.replace(key, value) == null) {
```

Comment:
So this is a funky one, probably should have commented the thought process. replace() only does anything if the key is there. If there's nothing, then it returns null. The idea here is that if we run into a key that we've already seen, we just overwrite the value. If we can't, then it's a new one, and we want to use put().